src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Export/GroupMemberExtractor.cs

```diff
@@ -88,7 +88,11 @@ private async Task<HashSet<string>> GetGroupPatientIdsHelper(string groupId, Dat
 
         public async Task<List<Tuple<string, string>>> GetGroupMembers(string groupId, DateTimeOffset groupMembershipTime, CancellationToken cancellationToken)
         {
-            var groupResource = await _fhirDataStore.Value.GetAsync(new ResourceKey(KnownResourceTypes.Group, groupId), cancellationToken);
+            ResourceWrapper groupResource;
+            using (IScoped<IFhirDataStore> dataStore = _fhirDataStoreFactory())
```

Comment:
nit: prefer Invoke, as it looks less like a method within the class:
```suggestion
            using (IScoped<IFhirDataStore> dataStore = _fhirDataStoreFactory.Invoke())
```

Reply:
Updated code to use `Invoke()`